WalletWasabi/Tor/TorProcessManager.cs

```diff
@@ -42,6 +42,11 @@ public TorProcessManager(TorSettings settings, EndPoint torSocks5EndPoint)
 
 		private bool _disposed = false;
 
+		public Task<bool> IsTorRunningAsync()
+		{
+			return TorSocks5Client.IsTorRunningAsync();
```

Comment:
ConfiguraAwait false

Reply:
Also you can use this method in this file instead of TorSocks5Client.IsTorRunningAsync()

Reply:
> ConfiguraAwait false

This method is not marked with `async` keyword, you can't use `ConfigureAwait` here. You can change it to:

```csharp
public async Task<bool> IsTorRunningAsync()
{    
    return await TorSocks5Client.IsTorRunningAsync().ConfigureAwait(false);
}
```

but that is only performance-wise slower, so I don't see a point.

Reply:
> Also you can use this method in this file instead of TorSocks5Client.IsTorRunningAsync()

Yes, done.

Reply:
I see you return the task itself, so far so good.

Reply:
TorProcessManager.cs  line 110 also using TorSocks5Client.IsTorRunningAsync()

Reply:
Ah, I did not notice.

Reply:
Fixed.